src/components/search/search.jsx

```diff
@@ -0,0 +1,29 @@
+/* eslint-disable max-len */
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Search = ({
+  search,
+  onChange,
+  onSubmit
+}) => (
+  <>
+    <label htmlFor="search">News Search</label>
+    <input
+      id="search"
+      type="search"
+      name="search"
+      value={search}
+      onChange={onChange}
+    />
+    <button onClick={onSubmit} data-testid="button">Search For Articles</button>
```

Comment:
This all belongs in a form with the onSubmit being for the form, not an onClick for the button to handle it